public/tabs/chat/chat_page.tsx

```diff
@@ -6,12 +6,11 @@
 import { EuiFlyoutBody, EuiFlyoutFooter, EuiPage, EuiPageBody, EuiSpacer } from '@elastic/eui';
 import React, { useCallback, useState } from 'react';
 import cs from 'classnames';
-import { useChatContext } from '../../contexts/chat_context';
-import { useChatState } from '../../hooks/use_chat_state';
+import { useObservable } from 'react-use';
```

Comment:
I am a little bit concerned about this shadow dependency. Should we declare the dependency of `react-use` in the package.json of dashboards-assistant?